content/components/selectpanel.mdx

```diff
@@ -9,51 +9,155 @@ import {Box} from '@primer/react'
+However, when you separate items in groups this option isn't available anymore.
+
+Additionally, It can only be used when all available options can be returned. As an example, if the list is too large to retrieve all options at once such as API endpoints returning limited results then this feature may not be suitable.
+We should refrain from putting a strain on API endpoints to return all the possible results at once as this could result in a performance downgrade in a scenario of large lists as virtualization is currently not built into this component.
+
+The `intermediate selections` option is useful for performing bulk actions on lists. For example, if you have 20 selected items, but some of them have different labels attached, an indeterminate checkbox (represented by a horizontal line) is shown for labels that are not set on all items.
+
+### Footer
+
+<img
+  width="1388"
+  alt="Footers"
+  src="https://github.com/primer/design/assets/980622/85bcb93b-3d39-4635-ad1b-68243dae6bd9"
+/>
+
+The optional footer at the bottom could include a single link, button or
+checkbox to take additional actions.
+
+In case of a multi selection or a single selection with additional actions in the footer, we always recommend a `cancel` and `apply` button unless you clearly [understand the accessibility implications](#cancelsubmit) of not adding this.
+
+### Current selection
+
+<img width="960" alt="" src="https://github.com/primer/design/assets/980622/b1c517f4-8cc0-423b-83ac-da348aec8e76" />
+
+In order to present a clear summary of the current selection, we create a group at the top of the list.
+This is visible when clearing the search input or when the select panel is reopened.
+However, it's crucial we don't create this group immediately after selecting an item, as it may cause reordering of the items and potentially confuse the users.
+
+### Error/Warning
+
+<img
+  width="1466"
+  alt="Error"
+  src="https://github.com/primer/design/assets/980622/135d597a-33cd-4ba1-a86b-241a1b5bdfcb"
+/>
+
+Sometimes, we encounter situations where things don't work as expected, or we need to inform users about something important. There are various ways to handle these scenarios depending on the situation.
+
+For instance, if the default values fail to load initially, we display an error placeholder to let the user know that something went wrong.
+
+When the initial values are loaded and the user performs a search for new values, there may be instances where the search fails. In such cases, it is essential to ensure that the user can still make a selection without losing their current selection. Instead, we display an error message at the top of the list, notifying the user about the search failure while preserving their ability to continue with their selection process.
+
+For errors we should always include a link to our support page to ensure that users have a way to reach out to us for assistance.
```

Comment:
I'm not sure about this recommendation. There may be errors that support can't do anything about, and it would be a waste of the user's time to contact support.